schutzbot/deploy.sh

```diff
@@ -92,6 +92,7 @@ echo -e "fastestmirror=1" | sudo tee -a /etc/dnf/dnf.conf
 
 # TODO: include this in the jenkins runner (and split test/target machines out)
 sudo dnf -y install jq
+sudo pip install yq
```

Comment:
We can move this into`test/cases/ostree-simplified-installer.sh` because it's required by simplified installer test script only.

Reply:
thanks! fixed